tests/Microsoft.IdentityModel.Protocol.Extensions.Tests/CrossTokenTests.cs

```diff
@@ -84,6 +84,17 @@ public void CrossToken_ValidateToken()
             ClaimsPrincipal jwtPrincipal = ValidateToken(jwtToken, IdentityUtilities.DefaultAsymmetricTokenValidationParameters, jwtHandler, ExpectedException.NoExceptionExpected);
             ClaimsPrincipal jwtPrincipal2 = ValidateToken(nullIssuerJwtToken, IdentityUtilities.GetNullIssuerAsymmetricTokenValidationParameters(false), jwtHandler, ExpectedException.NoExceptionExpected);
             ClaimsPrincipal jwtPrincipal3 = ValidateToken(nullIssuerJwtToken, IdentityUtilities.GetNullIssuerAsymmetricTokenValidationParameters(true), jwtHandler, ExpectedException.SecurityTokenInvalidIssuerException("IDX10211:"));
+
+            jwtHandler.SetDefaultTimesOnTokenCreation = false;
+            string nullLifetimeJwtToken = IdentityUtilities.CreateJwtToken(IdentityUtilities.NullLifetimeAsymmetricSecurityTokenDescriptor, jwtHandler);
```

Comment:
Do we have a test that checks that the 'exp' and 'nbf' claims in the JwtPayload are null?